src/item-inspector/files-by-type/ia-files-by-type.ts

```diff
@@ -0,0 +1,100 @@
+import { MetadataResponse, File } from '@internetarchive/search-service';
+import { html, customElement, LitElement, property } from 'lit-element';
+
+interface CollapsedFileInt extends File {
+  [menuId: string]: any;
+  derivatives: [];
+}
+
+@customElement('ia-files-by-type')
+export default class IaFilesByType extends LitElement {
+  @property({ type: Object }) item: MetadataResponse = {} as MetadataResponse;
```

Comment:
I would recommend starting off with `undefined` instead of casting an empty object into a response.